test/e2e/destroypod_test.go

```diff
@@ -36,7 +36,7 @@ import (
 
 const (
 	timeoutExpectedOutput  = "Slept for 0 milliseconds"
-	revisionTimeout        = 45 * time.Second
+	revisionTimeout        = 45
```

Comment:
nit: I feel like we should leave these "user-facing" values timeouts and only pass them properly broken down (to seconds) when time comes. Makes them much more readable.

Reply:
I'm not sure I understand what you want here?